### PR TITLE
Removed overlap on Facility Page in mobile view

### LINKED ISSUE #3124: Text is going outside on Facility page in mobile view
https://care.coronasafe.in/facility/df1a22fb-1875-4949-af85-15d1917b8bfd

image

### PR DESCRIPTION
**Pull request by Pranshu1902**
Fixes #3124 
http://localhost:4000/facility/f8f05c04-5c97-415c-814d-419ecf692d0e

image

### CODE REVIEW

File: src/Components/Facility/FacilityHome.tsx [hunk 1]
```diff
@@ -297,7 +300,7 @@ export const FacilityHome = (props: any) => {
                         {facilityData?.local_body_object?.name}
                       </p>
                     </div>
-                    <div className="flex gap-4">
+                    <div className="flex flex-col md:flex-row lg:flex-row gap-4">
```
Comment (developedBySJ) [inline src/Components/Facility/FacilityHome.tsx]: `lg:flex-row` not needed

**Review by developedBySJ — CHANGES_REQUESTED** [review]

File: src/Components/Facility/FacilityHome.tsx (hunk 1 above)
Comment (Pranshu1902) [reply src/Components/Facility/FacilityHome.tsx]: Done

**Review by kunatastic — APPROVED** [review]

**Review by cp-Coder — APPROVED** [review]

**Review by developedBySJ — APPROVED** [review]
LGTM

**Comment by nihal467** [comment]
@Pranshu1902 
image

make it responsive

**Comment by Pranshu1902** [comment]
@nihal467 
Updated view:
image

**Comment by nihal467** [comment]
@Pranshu1902 
image

still issue

**Comment by Pranshu1902** [comment]
@nihal467 
Updated view:
image

**Comment by nihal467** [comment]
test approved

**Review by khavinshankar — APPROVED** [review]

**Pull request merged**